sdcm/nemesis.py

```diff
@@ -250,14 +242,27 @@ def repair_nodetool_rebuild(self):
         self._run_nodetool(rebuild_cmd, self.target_node)
 
 
-def log_time_elapsed(method):
+def log_time_elapsed_and_status(method):
     """
     Log time elapsed for method to run
 
     :param method: Remote method to wrap.
     :return: Wrapped method.
     """
+    def print_nodetool_status(self):
+        for node in self.cluster.nodes:
```

Comment:
How about check the nodes number here (before and after nemesis)?
The number should not be changed here.